miden/src/cli/verify.rs

```diff
@@ -44,8 +44,12 @@ impl VerifyCmd {
         // verify proof
         verifier::verify(
             program_hash,
-            &input_data.stack_inputs,
-            &outputs_data.outputs,
+            &input_data.stack_init,
+            &(outputs_data
+                .outputs
+                .iter()
+                .map(|v| v.parse::<u64>().unwrap())
+                .collect::<Vec<u64>>()),
```

Comment:
I'd probably put parsing of strings into integers into `OutputFile` struct (maybe exposed via a separate method). This way, all parsing logic is encapsulated there.